#5: Added the readme.md file. (#7)
RonasIT/laravel-media: readme.md

# Laravel Media plugin

## Introduction

This plugin adds the ability for users to work with media files.

## Installation

1. Install the package using the following command: `composer require ronasit/laravel-media`
2. Run `php artisan vendor:publish`
3. For Laravel <= 5.5 add `RonasIT\Media\MediaServiceProvider::class` to config `app.providers` list.

> 4. In `media.permitted_types` config, specify the allowed types of media files.

Reviewer: ```suggestion
```

Author: Done.

> 5. In `media.classes.user_model` config, specify the model class that will have a relationship with the media class.

Reviewer: ```suggestion
4. Set your project's User model to the `media.classes.user_model` config.
```

Author: Done.

> 

Reviewer: ```suggestion

## Integration with [LaravelSwagger](https://github.com/RonasIT/laravel-swagger)

This package includes OpenAPI documentation file. To include it to your project's documentation, you need to register it in the `auto-doc.additional_paths` config:

`vendor/ronasit/laravel-media/documentation.json`
```

Author: Done.

Reviewer: nope, please check empty lines in the suggestion

Author: Fixed.

# Laravel Media plugin

## Introduction

This plugin adds the ability for users to work with media files.

## Installation

1. Install the package using the following command: `composer require ronasit/laravel-media`
2. Run `php artisan vendor:publish`

> 3. Add `src/MediaServiceProvider` to `config/app.php` your application

Reviewer: this step requires for the Laravel < 5.5

Author: Fixed.

### Configs

- `media.permitted_types` - list of allowed media file types.

> - `media.classes.user_model` - user model class that will have a relationship with the media class.

Reviewer: Let's remove configs block from the readme and move user configuration step to the installation block

Author: Done.
